plugins/aws/cloudtrail/cloudtrailObjectLock.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
             }
 
             if (!describeTrails.data.length) {
-                helpers.addResult(results, 0, 'No trails found', region);
+                helpers.addResult(results, 2, 'CloudTrails is not enabled', region);
```

Comment:
actually, it will mess up the tests, can you revert this to No trails found, it sounds better.

Reply:
I had changed the test as well. The thing is that in all other CloudTrail plugins, we are displaying CloudTrail is not enabled. So I changed it to make it consistent. Do you still want me to change it back?